context/src/env/validator.rs

```diff
@@ -244,24 +259,48 @@ pub fn validate(ci_info: &CIInfo) -> EnvValidation {
     env_validation
 }
 
+#[cfg_attr(feature = "pyo3", pyclass)]
+#[cfg_attr(feature = "wasm", wasm_bindgen)]
 #[derive(Debug, Clone, Default, PartialEq, Eq)]
 pub struct EnvValidation {
     level: EnvValidationLevel,
     issues: Vec<EnvValidationIssue>,
 }
 
+#[cfg_attr(feature = "pyo3", pyclass(get_all))]
+#[cfg_attr(feature = "wasm", wasm_bindgen(getter_with_clone))]
+#[derive(Debug, Clone, Default, PartialEq, Eq)]
+pub struct EnvValidationFlatIssue {
+    pub level: EnvValidationLevel,
+    pub error_message: String,
+}
+
+#[cfg_attr(feature = "pyo3", pymethods)]
+#[cfg_attr(feature = "wasm", wasm_bindgen)]
 impl EnvValidation {
     pub fn level(&self) -> EnvValidationLevel {
         self.level
     }
 
-    pub fn issues(&self) -> &[EnvValidationIssue] {
-        &self.issues
+    pub fn issues_flat(&self) -> Vec<EnvValidationFlatIssue> {
+        self.issues
+            .iter()
+            .map(|i| EnvValidationFlatIssue {
+                level: EnvValidationLevel::from(i),
+                error_message: i.to_string(),
+            })
+            .collect()
```

Comment:
Python and wasm don't work with enums that have nested (tuple or struct variants), so flattening can be done to essentially give the same data. Also, references cannot be passed back and forth, so owned data is returned instead.